server/src/main/java/com/testsigma/controller/TestStepsController.java

```diff
@@ -98,10 +98,7 @@ public TestStepDTO create(@RequestBody TestStepRequest request) throws Testsigma
   public void bulkDelete(@RequestParam(value = "ids[]") Long[] ids) throws ResourceNotFoundException {
     // [TODO] [Pratheepv] position update is stopping from bulk delete query
     log.debug("DELETE /test_steps/bulk_update_properties with ids::" + Arrays.toString(ids));
-    for (Long id : ids) {
-      TestStep step = this.service.find(id);
-      this.service.destroy(step);
```

Comment:
Why is destroy removed ? Why is bulkDelete method required ?

Reply:
this part is moved to teststep service

Reply:
Ok